File: app/src/main/java/eu/se_bastiaan/marietje/data/model/Empty.java
package eu.se_bastiaan.marietje.data.model;

import com.google.auto.value.AutoValue;
import com.google.gson.Gson;
import com.google.gson.TypeAdapter;

@AutoValue
public abstract class Empty {

    public static Empty create() {
        return new AutoValue_Empty();
    }

    public static TypeAdapter<Empty> typeAdapter(Gson gson) {
        return new AutoValue_Empty.GsonTypeAdapter(gson);
    }

}

File: app/src/main/java/eu/se_bastiaan/marietje/data/model/Empty.kt
package eu.se_bastiaan.marietje.data.model

import android.os.Parcel
import android.os.Parcelable
import paperparcel.PaperParcel

@PaperParcel
class Empty : Parcelable {
    companion object {
        @JvmField val CREATOR = PaperParcelEmpty.CREATOR
    }

    override fun describeContents() = 0

    override fun writeToParcel(dest: Parcel, flags: Int) {
        PaperParcelEmpty.writeToParcel(this, dest, flags)
    }
}